src/utils/include/utils/Array.hpp

```diff
@@ -26,6 +26,8 @@
 
 #include <cassert>
 #include <cstddef>
+#include <iostream>
```

Comment:
Isn't `ostream` sufficient? `iostream` brings in initialization code and a global state.

Reply:
Right.